doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -399,6 +399,16 @@ The following strings must not be used as attribute names:
 
 * Virtual attribute names (see specific section on "Virtual Attributes")
 
+## Metadata names restrictions
+
+The following strings must not be used as metadata names:
+
+* `previousValue`, as it is a reserved metadata (see "Special metadata in notifications" section).
+
+* `actionType`, as it is a reserved metadata (see "Special metadata in notifications" section).
+
+* `*`, as it has an special meaning as "all the custom (user) metadata".
```

Comment:
Good point. However, as long as `*` is a valid ASCII char and it is not a control character, whitespace, `&`, `?, `/` or `#`, it fits the name requirements.